### ISSUE TITLE
Control animations/duration

### ISSUE THREAD

**Issue by ABLomas** [LycheeOrg/Lychee-Laravel#128]
TL;DR - can we have photo display animations configurable?

Currently we have nice animations on delete, move. Also nice "blackout" effect on next/prev. photo.
It's ok when we have few hundreds pix. But when there are thousands of them, effects don't add much value.
I tried to find where lychee.animate function gets it's delay but no luck (spent not so much time, i will debug more). 
Maybe we can have simple/no animation effects, like fancybox "none" or something like that? Currently if you just move trough photos forward/back it's blinking ("zooming") a lot and i already know at least two ppl who won't watch because they are very sensitive to such blinking.

**Comment by ildyria** [comment]
@SerenaButler might have some hint for you.

Also this should work: https://github.com/LycheeOrg/Lychee/wiki/FAQ#how-to-disable-the-zoom-animation-while-browsing-pictures

**Comment by ildyria** [comment]
Additionally having a setting for that is quite difficult to set up. :/

**Comment by ABLomas** [comment]
Thank you! Was looking for FAQ on link which is bottom of this page: https://github.com/LycheeOrg/Lychee-Laravel - but it points to non-existing page, so i decided that FAQ does not exists - and should be checking Lychee FAQ for that.
Looks good, without animations it's much better, thanks!
(1000+ photos and growing fast, should import ~1K more photos, will see how it's going ;-)

**Comment by ildyria** [comment]
I'm running Lychee-Laravel with 4000 pictures :+1:  :smile:
